sites/friday/src/get-started/here/make-changes.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Make changes to your project
+sidebarTitle: Revisions
+weight: 50
+description: How to work on a daily basis with an {{% vendor/name %}} project?
+---
+
+Upsun allows you to make changes to your project and test them on a preview environment before introducing them to Production.
+
+In your project, the default branch (e.g. `main`, `master`, whichever chosen during project creation) always represents the production environment.
+Other branches are for developing new features, fixing bugs, or updating the infrastructure.
+
+To make changes to your project, follow these steps:
+
+## 1. Create a new environment
+Create a new environment (a Git branch) to make changes without impacting production:
+{{< codetabs >}}
++++
+title={{% vendor/name %}} Git repository
++++
+
+Run the following command:
+   ```bash {location="Terminal"}
+   {{% vendor/cli %}} branch feat-a
+   ```
+   This creates a new local `feat-a` Git branch based on the `main` Git branch
+   and activates a related environment on {{< vendor/name >}}.
+   The new environment inherits the data (service data and assets) of its parent environment (the production environment here).
+<--->
++++
+title=Using third party provider
++++
+   {{% vendor/name %}} provides a feature called [source integration](integrations/source.html) that allows your {{% vendor/name %}} project to be fully integrated with your external repository.
+   This enables you, as a developer, to use a normal Git workflow (`git add . && git commit -m "message" && git push`) to deploy your environment—with no need to connect to the {{% vendor/name %}} Console.
+
+   Assuming you've already activated the source integration feature on your project, create a new branch:
+   ```bash {location="Terminal"}
+   git checkout -b feat-a && git push -u origin feat-a
+   ```
+   This creates a new local `feat-a` Git branch based on the `main` Git branch
+   and a related environment on {{< vendor/name >}}.
+   The new environment is inactive by default. Once you activate it, it will inherit the data (service data and assets) of its parent environment (the Production environment here).
+
+   {{< note >}}
+   When your local feature is ready, [activate your environment](/administration/cli/reference.html#environmentactivate).
+   {{< /note >}}
+
+{{< /codetabs >}}
+
+{{< note title="Warnings" theme="warning">}}
+   **First**, you are only charged for your [activated environments](/administration/cli/reference.html#environmentactivate).
+   So, to create an inactive environment waiting for you to be ready to test, run the following command:
+   ```bash {location="Terminal"}
+   git branch checkout -b feat-a && git push -u upsun feat-a
+   ```
+   **Secondly**, if you have set up a [source integration](/integrations/source/_index.md), make sure you disable the `prune_branches` option (`true` by default). Otherwise, the `upsun branch` command fails at the end of the deployment.
```

Comment:
if we keep the split path between `Upsun Git repository` and `Using third party provider`, we should move this second warning to the bottom of the `Using third party provider`section just after we have them activate their environment.

Reply:
i moved it inside the third party provider tab but now, i feel uncomfortable with making this warning hidden at first look (need to select the third party provider to read it). 
https://docs.upsun.com.fix-generic-gstarted-7jc6g4y-ucq44jg6ofare.eu-5.platformsh.site/get-started/here/make-changes.html#1-create-a-new-environment